Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -151,7 +151,15 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
             this._basicPropertyRowsMap.set(propertyName, this._createDetailsSectionRowForProperty(propertyName));
         }
 
-        this._basicPropertiesGroup.rows = [...this._basicPropertyRowsMap.values()];
+        this._basicPropertiesGroup.rows = [...this._basicPropertyRowsMap.values()].sort((rowA, rowB) => {
```

Comment:
Sorts the basic properties rows into two categories: read-only first, editable last.
This way, the ones with editing controls are visually closer to the ones in the _Variation Properties_ section, if present.